lib/libc/pthread/pthread_mutexattr_setprotocol.c

```diff
@@ -80,25 +80,26 @@
  *
  ****************************************************************************/
 
-int pthread_mutexattr_setprotocol(FAR pthread_mutexattr_t *attr,
-				  int protocol)
+int pthread_mutexattr_setprotocol(FAR pthread_mutexattr_t *attr, int protocol)
 {
 	svdbg("attr=0x%p protocol=%d\n", attr, protocol);
-	DEBUGASSERT(attr != NULL);
```

Comment:
If it is required , I can remove my change.
As I am doing the task of code coverage where we require (code coverage>80%) , I replaced this statement with an if() statement to test one test case where attr=NULL.